frequency           : 20    # [hz]

pc_raw_topic        : /velodyne_points
pc_ground_topic     : /velodyne_points/ground
pc_obstacle_topic   : /velodyne_points/obstacle

sensor_height       : 0.85        # [m]

radius_coeff_close  : 0.2
radius_coeff_far    : 0.7

max_slope           : 10          # [deg]

point_num_min       : 2           # []

gap_threshold       : 0.1         # [m]